server/server.js

```diff
@@ -18,7 +18,6 @@ app.use(express.static(path.join(__dirname, '/../client/dist')))
 
 // app.use('/overview', overview);
 app.use('/relatedProducts', relatedProducts);
-// app.use('/reviews', reviews);
```

Comment:
Thanks for explaining in zoom meeting! We can make this all through one request or just a call to for the info!